packages/tooltips/src/views/LightTooltip.js

```diff
@@ -70,7 +70,7 @@ LightTooltip.propTypes = {
 
 LightTooltip.defaultProps = {
   arrow: true,
-  size: SIZE.LARGE
```

Comment:
Alternatively, the default `size` prop of both light and dark tooltip could be updated to be `SIZE.MEDIUM`.